wce-ote/plugin/plugin.js

```diff
@@ -2545,18 +2545,19 @@
 			}
 
 			var langEn = language.substring(0, 2) == "en";
+			var ignoreShiftNotEn = tinyMCE.activeEditor.settings.ignoreShiftNotEn ? tinyMCE.activeEditor.settings.ignoreShiftNotEn : [];
```

Comment:
I was torn between moving ignoreShiftNotEn to clientOptions.  I had intended to do it, but after looking at your PR, I like how you have something higher level in clientOptions (transcriptionLanguage) and we can handle the lower level stuff.  I guess if someone wanted to support another language, they could provide additional character stokes they wanted to bypass the special OTE keyboard handling.  I guess that would lean toward moving it to clientOptions.  I still like the distinction of keeping our added options (ignoreShiftNotEn) out of the tinyMCE.settings object.  But I see the benefit of moving it in and also keeping it out.  But all in all, I so grateful you've taken the time to merge this into the core code.  This has been on my to-do list for years.  Thank you!

Reply:
I don't have any strong feelings about this. I didn't realise it was something added by 'us' and not already an option in Tinymce which is why I didn't even consider moving it. It will be easy to move again later if we choose to and I think that would also be fine. We could still have a single higher level clientOptions setting which set other settings in clientOptions if they are not supplied separately as clientOptions. I will add a ticket to revisit this later (there are a few like that where I did 'good enough for now' fixes that I think would be worth looking at again when we have more time).